### ISSUE TITLE
[QUESTION] How to choose between multiple pritunl servers?

### ISSUE THREAD

**Issue by jserpapinto** [nathanielvarona/pritunl-client-github-action#80]
We have a profile with access to several pritunl "servers", is it possible to choose one of them in the github action?

Thanks!

**Comment by nathanielvarona** [comment]
Hello @jserpapinto,

Thank you for raising this helpful issue.

Right now, we don't have the support for selecting a specific server in a single profile.

In the current version, I use a `raw text table` to filter the profile, and it is broken when multiple server is attached to a profile. Example:
```
+------------------+--------------------------------+----------+-----------+--------------+----------------+----------------+
|        ID        |              NAME              |  STATE   | AUTOSTART |  ONLINE FOR  | SERVER ADDRESS | CLIENT ADDRESS |
+------------------+--------------------------------+----------+-----------+--------------+----------------+----------------+
| eccrgm6dzbilpvzt | pritunl.profile                | Inactive | Enabled   | Disconnected | -              | -              |
|                  | (pritunl-dev-1)                |          |           |              |                |                |
| m2py0tvseom30mql | pritunl.profile                | Inactive | Enabled   | Disconnected | -              | -              |
|                  | (pritunl-dev-2)                |          |           |              |                |                |
+------------------+--------------------------------+----------+-----------+--------------+----------------+----------------+
```

I will release version `v1.3.0` that uses JSON outputs to filter the server we want easily. Example:
```json
[
  {
    "id": "eccrgm6dzbilpvzt",
    "name": "pritunl.profile (pritunl-dev-1)",
    "state": "Enabled",
    "run_state": "Inactive",
    "registration_key": "",
    "connected": false,
    "uptime": 0,
    "status": "Disconnected",
    "server_address": "",
    "client_address": ""
  },
  {
    "id": "m2py0tvseom30mql",
    "name": "pritunl.profile (pritunl-dev-2)",
    "state": "Enabled",
    "run_state": "Inactive",
    "registration_key": "",
    "connected": false,
    "uptime": 0,
    "status": "Disconnected",
    "server_address": "",
    "client_address": ""
  }
]

```

On our GitHub Actions, a new option will be added the `profile-server`. Example:
```yml
- name: Setup Pritunl Profile and Start VPN Connection
  uses: nathanielvarona/pritunl-client-github-action@v1
  with:
    profile-file: ${{ secrets.PRITUNL_PROFILE_FILE }}
    ...
    ...
    profile-server: 'pritunl-dev-2'
    ## Using the JSON key 'name' -- 'PROFILE_NAME (SERVER_NAME)'
```

Right now, the easiest solution is creating a `new user and organization` and attaching it to the specific `server` you want to use.

I will soon post the update for the release with the option `profile-server.` 

Thanks,

**Comment by nathanielvarona** [comment]
Hello again @jserpapinto 

Another option to connect to a specific server of a profile is using the manual connection. Example:

## Baisc

```yml
- name: Setup Pritunl Profile and Start VPN Connection
  uses: nathanielvarona/pritunl-client-github-action@v1
  with:
    profile-file: ${{ secrets.PRITUNL_PROFILE_FILE }}
    start-connection: 'false'

    
- name: Starting a VPN Connection Manually for a Specific Server of a Profile
  shell: bash
  run: |
    pritunl-client list --json |
      jq '.[] | select(.name | contains("pritunl-dev-2"))' |
      jq '.id' |
      xargs pritunl-client start
```

## With other options

```yml
- name: Starting a VPN Connection Manually for a Specific Server of a Profile
  shell: bash
  run: |
    pritunl-client list --json |
      jq '.[] | select(.name | contains("pritunl-dev-2"))' |
      jq '.id' |
      xargs pritunl-client start --password ${{ secrets.PRITUNL_PROFILE_PIN }}
```

Thanks,

**Comment by nathanielvarona** [comment]
Hello again, @jserpapinto,

[v1.3.0](https://github.com/nathanielvarona/pritunl-client-github-action/releases/tag/v1.3.0)+ just released. You can now have the option to choose your server for a profile with multiple servers.

Example:
```yaml
- name: Setup Pritunl Profile and Start VPN Connection
  uses: nathanielvarona/pritunl-client-github-action@v1
  with:
    profile-file: ${{ secrets.PRITUNL_PROFILE_FILE }}
    profile-server: 'pritunl-dev-2'
```
Docs: https://github.com/nathanielvarona/pritunl-client-github-action#if-the-profile-has-multiple-servers-and-want-to-specify-one

Tests Results:
[![Connection Tests - Complete](https://github.com/nathanielvarona/pritunl-client-github-action/actions/workflows/connection-tests-complete.yml/badge.svg?branch=main)](https://github.com/nathanielvarona/pritunl-client-github-action/actions/workflows/connection-tests-complete.yml?query=branch:main) 
[![Connection Tests - Basic](https://github.com/nathanielvarona/pritunl-client-github-action/actions/workflows/connection-tests-basic.yml/badge.svg?branch=main)](https://github.com/nathanielvarona/pritunl-client-github-action/actions/workflows/connection-tests-basic.yml?query=branch:main)

Enjoy,

**Comment by nathanielvarona** [comment]
The feature is available on [v1.3.0](https://github.com/nathanielvarona/pritunl-client-github-action/releases/tag/v1.3.0)+ Releases, closing this issue.

**Comment by jserpapinto** [comment]
Hey @nathanielvarona, thanks a lot for the quick support and changes!

Appreciated!
